yarn-project/end-to-end/src/e2e_2_pxes.test.ts

```diff
@@ -249,6 +249,9 @@ describe('e2e_2_pxes', () => {
       note = notes[0];
     }
 
+    // TODO(#12013): We need to do this hack because NoteDao no longer populates noteTypeId
+    note.noteTypeId = TestContract.notes.ValueNote.id;
```

Comment:
I am not really sure why this got triggered now as even before the noteTypeId should have been non-zero but now it caused the test to fail when adding the nullified note.

Given that note type id will be soon nuked from NoteDao and ExtendedNote and given that we will nuke the `addNullifiedNote` function doesn't make sense to bother with.